docs/community_resources/installing-on-mac-os-and-windows.md

```diff
@@ -117,6 +117,7 @@ The configuration requires providing the following parameters:
 - `DATABASE_PORT`
 - `DATABASE_PLATFORM` —  prefix for distinguishing the database you are connecting to (e.g. `mysql` or `pgsql`)
 - `DATABASE_DRIVER` — driver used by Doctrine to connect to the database (e.g. `pdo_mysql` or `pdo_pgsql`)
+- `DATABASE_VERSION` - database server version (for a MariaDB database, prefix the value with `mariadb-`)
```

Comment:
IMHO we should link to original article at some point: https://symfony.com/doc/current/reference/configuration/doctrine.html#doctrine-dbal-configuration